interfaces/builtin/kvm.go

```diff
@@ -38,8 +48,60 @@ const kvmConnectedPlugAppArmor = `
 
 var kvmConnectedPlugUDev = []string{`KERNEL=="kvm"`}
 
+type kvmInterface struct {
+	commonInterface
+}
+
+var procCpuinfo = "/proc/cpuinfo"
+
+func getCpuFlags() (flags string, err error) {
+	buf, err := ioutil.ReadFile(procCpuinfo)
+	if err != nil {
+		// if we can't read cpuinfo, we want to know _why_
+		return "", fmt.Errorf("error: %v", err)
+	}
+
+	pattern := []byte("\nflags\t\t:")
+	idx := bytes.LastIndex(buf, pattern)
+	if idx >= 0 {
+		offset := idx + len(pattern)
+		endidx := bytes.Index(buf[offset:], []byte("\n"))
```

Comment:
This can return `-1`, it's not super likely but we should check for it.

Reply:
Replaced with regexp, and function returns []string, this is gone